D4e/d4e_functions.py

```diff
@@ -70,27 +68,22 @@ async def attack(
         roll = roll_list[1]
 
     # Roll the dice
-    if attack_modifier != "":
-        if attack_modifier[0] == "+" or attack_modifier[0] == "-":
-            roll_string = roll + attack_modifier
-        else:
-            roll_string = roll + "+" + attack_modifier
-    else:
+    # print(roll)
+    try:
+        roll_string: str = f"{roll}{ParseModifiers(attack_modifier)}"
+    except:
         roll_string = roll
```

Comment:
We definitely want to have a rework on how we handle exceptions. But this is fine for the current implement.

Reply:
Moved to https://github.com/Artica2012/VirtualGM/issues/104

Reply:
Taken to: https://github.com/Artica2012/VirtualGM/issues/105